### ISSUE TITLE
root_numpy and conda?

### ISSUE THREAD

**Issue by arogozhnikov** [rootpy/root_numpy#223]
To start with, I'm developing REP, which wraps different classifiers.
`root_numpy` is important part we're relying on both to work with ROOT data and handle TMVA.

Though I really like it, this is the most problematic dependency (due to ROOT). 
I'm looking for not-so-slow and reliable way to deploy REP (Docker is definitely bad option for trying something new). 

Up to this moment the only way I found which is able to work not only on Ubuntu is miniconda.

Probably, you know that there is painless way to use root_numpy:
```
conda install -c nlesc root_numpy
```

This works in Ubuntu (I checked) and claimed to work at Mac OS, but not windows.

This method I knew from @ibab,  and I believe that package is prepared by @remenska.

I am able to get all other dependencies of REP using conda:
https://travis-ci.org/arogozhnikov/rep/jobs/86099983

...so the question 
1. is it possible at all to prepare windows package for `root_numpy`?
2. if yes, does anyone ready to work on this?

This would terribly simplify everything (and I believe that not only for me, but for other users and developers too).

**Comment by remenska** [comment]
Hi @arogozhnikov 

windows package for root_numpy can be made available on conda, but again, it depends on ROOT. 
If you're able to provide a windows ROOT package on conda, problem would be solved. 

It's really a low priority at the moment, since I have no experience with compiling ROOT on Windows.

**Comment by ndawe** [comment]
I also have no experience with ROOT/root_numpy on Windows.

**Comment by arogozhnikov** [comment]
> I also have no experience with ROOT/root_numpy on Windows.

Neither do I. Tried digging.

The way proposed to build ROOT is MSVS (sic!) 2010.
This is not automatable at all. 

Since ROOT is compiled already for windows, one should skip this process and use compiled ROOT.

But the latest releases even don't have binaries for Windows (?!).

**Comment by kratsg** [comment]
I seriously doubt the ROOT team is focused on getting ROOT out for Windows -- there are still minor issues with Linux and Mac. Is there a particular reason one needs to run ROOT with Windows (I don't have much experience but it has been quite painful).

**Comment by ndawe** [comment]
I think this ends up as an issue on the ROOT side. They need to provide Windows support before we can do anything with root_numpy there. There are ROOT 5 builds for Windows but nothing for ROOT 6 yet.

https://root.cern.ch/phpBB3/viewtopic.php?t=18232

If windows support isn't essential, then you already have everything you need for Ubuntu and Mac OS, right? I'll close this for now and reopen later if needed.
